wallet/lib/stores/wallet_store_imp.dart

```diff
@@ -275,6 +275,9 @@ class WalletsStoreImp implements WalletsStore {
 
   @override
   Future<SdkIpcResponse> executeRecipe(Map json) async {
+    if (wallets.value.isEmpty) {
```

Comment:
In which case this error comes?

Reply:
> No unit test are still added to test this logic. @faizhashmiRnS make sure it does this. Otherwise i will close this PR.

@ahmadrns Please share the status here.

Reply:
When we implemented WalletStoreImp instead of MockWallet then this error comes in but we shifted all that to another branch because we have to change a lot of things in the code because of test cases, So removing it for now.

> No unit test are still added to test this logic. @faizhashmiRnS make sure it does this. Otherwise i will close this PR.

executeRecipe when user buying NFT & buy pylons both the test cases are already added. one in `wallet_store_imp_test.dart` named `should verify the recipe execution` & other in `repository_test.dart` named `test for adding pylons coins` just like all other test cases are implemented